idaes/models_extra/power_generation/costing/power_plant_costing.py

```diff
@@ -181,6 +525,11 @@ def get_PP_costing(self, cost_accounts, scaled_param, units, tech, ccs="B"):
 
     CE_index = fs.costing.CE_index
 
+    if CE_index_base is None:
+        CE_index_base = 671.1
```

Comment:
Not critical, it may be cleaner to use this value as the default in the method definition above instead of overwriting it here, i.e. `def get_PP_costing(self, cost_accounts, scaled_param, units, tech, ccs="B", CE_index_base=671.1):`